autorest/azure/environments.go

```diff
@@ -116,6 +117,7 @@ var (
 			Datalake:            "https://datalake.azure.net/",
 			Batch:               "https://batch.core.windows.net/",
 			OperationalInsights: "https://api.loganalytics.io",
+			OSSRDBMS:            "https://ossrdbms-aad.database.windows.net",
```

Comment:
FWIW, I am not sure `OSSRDBMS` is the right terminology here. This is really the active directory resource(?) URL to provide to AAD when requesting a token with access to MySQL. (Note that I am not 100% sure the terminology I used is correct either...)

See [connecting to azure database for mysql using azure AD](https://docs.microsoft.com/en-gb/azure/mysql/howto-configure-sign-in-azure-ad-authentication#connecting-to-azure-database-for-mysql-using-azure-ad).

It might be better to just say `MySQL` here?

Reply:
This ResourceID is also used for Postgres, Azure SQL, etc.

e.g. https://docs.microsoft.com/en-us/azure/postgresql/howto-connect-with-managed-identity#retrieving-the-access-token-from-azure-instance-metadata-service

The Azure CLI also uses this terminology:

```
lekonia:azure-service-operator.git{cloud-centric-environment-config} branche$ az cloud show
{
  "endpoints": {
...
    "ossrdbmsResourceId": "https://ossrdbms-aad.database.windows.net",
...
```

Reply:
This sounds good to me then! Thanks for clarifying.